index.js

```diff
@@ -0,0 +1,33 @@
+const Item = {
+  ItemId: 'owh12845-1kk2-9384-ckd0-akeiwh4968we',
+  name: 'Griff's leash',
+  description: 'The leash of the greatest pup in the world!',
+  price: 100000.99,
+  size: 'Griff sized'
+  };
+  
+const FirstName = 'Griffingford';
+const LastName = 'Qwerty';
+  
+const Customer = {
+  FirstName,
+  LastName,
+  email: '${FirstName.toLowerCase()}.${LastName.toLowerCase()}@drake.edu',
+  PhoneNumber: '+16439651289'
+};
+
+const Cart = {
+  Customer: FirstName LastName,
+  CreatedDate: 'March 14th 2019',
+  PuchasedDate: 'January 20th 2020'
+};
+
+const CartItem = {

```

Comment:
cart items will be a list and each cart item will be part of a single cart so they will share the same cartId
```suggestion
const CartItems= [
{
cartid: 'c573f256-50d7-11ea-8d77-2e728ce88125',
cartItemId: '4b6c8cfe-512a-11ea-8d77-2e728ce88125',
itemid:'b7671ac6-50d7-11ea-8d77-2e728ce88125',
quantity: 3
},
{
cartid: 'c573f256-50d7-11ea-8d77-2e728ce88125',
cartItemId: '5b6c8cfe-512a-11ea-9d66-2e728ce88135',
itemid:'c7761ac6-40d7-11eb-8e77-2d728ce88124',
quantity: 2
}
];

```